server/db/models/order.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cartItem = new mongoose.schema(
+	qty: Number,
+	product: { type: Schema.Types.ObjectId, ref: 'Product' },
+	color: String
```

Comment:
I don't think we'd have a color specification, a different color would be a different product id, no?